### DISCUSSION TITLE
part-ii/2018-6/

### DISCUSSION THREAD

**Discussion by tripos-comments** [tripos-education/maths-tripos-questions#54] [Question comments]
Part II 2018 Algebraic Topology: Paper 1, Section II, H 

https://questions.tripos.org/part-ii/2018-6/

**Comment by artie2000** [comment]
Does anyone have a nice map to start with for part (c)? I started with $\mathbb{R}^3\to\text{GL}_3(\mathbb{C})$, and it took ages to descend,

**Reply by eha1234** [reply]
<details><summary>An idea</summary>
 
Identify $S^1$ with $[0,1]$, then map $(t, (z,w)), t \in [0,1], (z,w) \in T$ to the matrix with $t, z, w$ in the 12, 13, 21 entries respectively. I'm not sure how to show this is a homeomorphism rigorously, but writing out the multiplication law of the matrices and comparing with how elements are identified with the gluing, it should work. 
</details>

**Reply by artie2000** [reply]
I think you need to have a - in the first coord

**Reply by ArthurConmy** [reply]
You show it's a homeomorphism by proving a cts bijection from compact space to Hausdorff space (Topological Inverse Function Theorem).
